index.js

```diff
@@ -1,46 +1,32 @@
 "use strict";
 
 const connoisseur = require("./lib/connoisseur");
+const fs = require("fs");
+const sourceFiles = require("yargs").argv._;
 
-const CODE = `
-"use strict";
-
-// lalala
-[4, 9, 16].map(Math.sqrt);  // → [ 2, 3, 4 ]
-const sth = function () { return "lalala" === 1; };  // asd
-sth();  // eslint-disable-line no-unused-expressions
-// → false
-/* eslint-disable no-unused-expressions */
-"test str";
-"test";  // → 'test'
-/* eslint-enable no-unused-expressions */
-const time = 3 * 15;  // eslint-disable-line no-unused-vars
-// endcomm
-`;
-
-const result = connoisseur(CODE, "stdin");
+const displayErrors = function (errors) {
+    let hasError = false;
+    errors.forEach(file => {
+        if (!file[1].length)
+            return;
 
-const exitCode = result.reduce((code, piece) => {
-    /* eslint-disable no-console */
-    let returnCode;
-
-    if (piece.error) {
-        returnCode = 2;
+        hasError = true;
+        /* eslint-disable no-console */
         console.log(
-            "Error: \n",
-            piece,
-            piece.error.stack
-                .split("\n")
-                .slice(0, 2)
-                .join("\n")
+            `Errors found for file ${file[0]}:\n${
+                file[1].reduce((prev, errorData) =>
+                    `${prev}\t${errorData.code.replace(/(\r\n|\n|\r)/gm, "\r\n\t")}\t${
```

Comment:
This is for fixing the display indentation of the multi-line codes. By the way I'm removing the `\r` 😅 but I'm not sure it will give correct output in a windows shell. Can you test it please?